src/pages/Bridge/BurnForm/index.tsx

```diff
@@ -158,7 +151,8 @@ const BurnForm = (): JSX.Element | null => {
         return 'Bridge must be loaded!';
       }
 
-      if (bitcoinAmountValue.to.Satoshi() === undefined) {
+      // Represents being less than 1 Satoshi
+      if (bitcoinAmountValue._rawAmount.lt(1)) {
```

Comment:
This could be a utility helper function as it's implicit.

Reply:
@peterslany 
Since I just ported the logic, I'm not sure as well but I think it does not mean zero.
@theodorebugnet @bvotteler @danhper @gregdhill @sander2 
Any comment on it?